src/app/personside/dialogpanel/sendMelding/SendNyMelding.tsx

```diff
@@ -80,7 +80,7 @@ function SendNyMelding(props: Props) {
 
     const erReferat = NyMeldingValidator.erReferat(state);
     const erSpørsmål = NyMeldingValidator.erSporsmal(state);
-
+    const erFormPabegynt = state.tekst || state.tema || state.sak;
```

Comment:
Denne tror jeg ikke er komplett.. Og kan kanskje løses på en litt mer elegant måte..?

Reply:
Ja begynte som en liten hack for å vise funksjonaliteten. Har du forslag på hvordan det kan løses bedre?